doc/modules/clustering.rst

```diff
@@ -1582,3 +1582,36 @@ Drawbacks
  *  Caliński, T., & Harabasz, J. (1974). "A dendrite method for cluster
     analysis". Communications in Statistics-theory and Methods 3: 1-27.
     `doi:10.1080/03610926.2011.560741 <http://dx.doi.org/10.1080/03610926.2011.560741>`_.
+
+.. _contingency_matrix:
+
+Contingency Matrix
+----------------------
+
+Contingency matrix is a format describing the relationship between
+discrete-valued random variables.
+It shows the multivariate frequency distribution of the variables.
+
+Let the first random variable be :math:`X`, and the second random
+variable be :math:`Y`.
+In a contingency matrix :math:`M`, entry :math:`M(i, j)` is the number of
+observations with :math:`X=X_i` and :math:`Y=Y_j`.
+
+Here is an example:
+
+
+	>>> from sklearn.metrics.cluster import contingency_matrix
+	>>> x = ["male", "male", "male", "female", "female", "female"]
+	>>> y = ["right_handed", "right_handed", "left_handed", "left_handed",
+	... "ambidextrous", "ambidextrous"]
+	>>> contingency_matrix(x, y) # doctest: +ELLIPSIS
```

Comment:
No need for the ELLIPSIS here

Reply:
I have a question regarding the use of `ELLIPSIS` https://stackoverflow.com/questions/47555297/doctest-ellipsis-use-in-rst-documentation-in-python

Reply:
ELLIPSIS is used when you are using `...` marker to match any substring:

https://docs.python.org/2/library/doctest.html#doctest.ELLIPSIS

Reply:
One more thing, in this section https://github.com/scikit-learn/scikit-learn/blob/master/doc/modules/clustering.rst#fowlkes-mallows-scores , for the block of code 'Bad (e.g. independent labelings) have zero scores:', which has an example returning `0.0` it also has `doctest: +ELLIPSIS` is that necessary?

Reply:
Probably not since that ... is not used. However, since it is there this does not required a PR :)